torch/testing/_internal/common_utils.py

```diff
@@ -881,6 +881,40 @@ def safeCoalesce(self, t):
 
         return tg
 
+    # Compares the given Torch and NumPy functions on the given tensor-like object.
+    #   if copy_np_result is True then NumPy arrays will be copied before being
+    #     compared with the torch tensor. This is useful if the NumPy array has
+    #     negative strides, which would cause the import to PyTorch to fail.
+    # TODO: support bfloat16 comparisons
```

Comment:
TODO: args/kwargs for torch, numpy, and assertEqual

Reply:
Maybe don't add args/kwargs for torch and numpy functions and suggest pattern of passing lambda (t) -> or partial function fn(t) -> instead